default/stringtables/en.txt

```diff
@@ -1210,6 +1210,12 @@ Enable Super-Tester Takeover
 RULE_ENABLE_SUPER_TESTER_DESC
 Enables [[BLD_SUPER_TEST]] building
 
+RULE_ALLOW_REPEATED_SPECIES
+Allows repeated species
+
+RULE_ALLOW_REPEATED_SPECIES_DESC
+Allows repeated species when there are less players than playable species
```

Comment:
Multiplying the number of species on 2 still may not fix this. 

Maybe we should change the code in a way that this expression will be true `len(specias_to_shoose) * n > len(players)`. Where n is multiplier from 1 to inf

Reply:
Species pool automatically populates while assign to empire:

https://github.com/freeorion/freeorion/blob/0590e073703eae52aaf6e8de2f33ebdb84cb0bf4/default/python/universe_generation/empires.py#L57-L62

Reply:
Got it.  The problem I described has already been solved. 

The description is still a bit confusing.

Maybe change it to
```
Allows repeated species, if the number of players is more than the number of species this option is turned on by default.
```

Reply:
I think current comments are pretty specific and unambiguous:

> fill the initial pool of playable species, without repetitions unless RULE_ALLOW_REPEATED_SPECIES is true

The initial pool of playable species won't have repeated species unless the game rule is 1.

Reply:
This is not about the comment in code, this is about string tables entry.

Reply:
Gah!

OK. Your suggestion seems also confusing to me. The option is not enabled when there are more players than species, it just refills itself when needed regardless of the option. So I tried a similar wording:

`Allows repeated species when there are less players than playable species. When there are more players than playable species there are always repeated species.`

The second sentence is redundant in my opinion, but I have no trouble leaving it there (it's only for the tooltip).

Reply:
> The second sentence is redundant in my opinion, but I have no trouble leaving it there (it's only for the tooltip).

The refill feature is documented in the code. So regular players cannot read it.

Reply:
So OK now?